portal/server/helpers/express-session.ts

```diff
@@ -0,0 +1,101 @@
+import { Request } from "express";
+import { PartiallyLoggedInSessionData, LoggedInSessionData, UserSessionData } from "../types/express-session";
+
+type Session = Request['session'];
+
+type LoggedInUserSession = Session & LoggedInSessionData;
+type PartiallyLoggedInUserSession = Session & PartiallyLoggedInSessionData;
+type UnknownLogInStatusUserSession = Session & Pick<UserSessionData, 'userToken' | 'loginStatus'>;
+
+/**
+ * By default, all session data will be optional
+ * (see use of `Partial` {@link https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/express-session/index.d.ts#L17 here})
+ *
+ * This helper function asserts that the `loginStatus`, `user` and `userToken` properties are
+ * present in the session, or throws an `Error` if not.
+ *
+ * Note: this doesn't assert on the validity of the user data values (which
+ * should have already been authenticated in middleware by this point), only
+ * that the values are present.
+ */
+export const asLoggedInUserSession = (session: Session): LoggedInUserSession  => {
+  const { user, userToken, loginStatus } = session as LoggedInUserSession;
+
+  if (loginStatus !== 'Valid 2FA' ) {
```

Comment:
`Valid 2FA` has been repeated multiple times in the micro-service, could benefit from a constant value please or perhaps a helper function which ascertain whether value is `Valid 2FA` or not.

Reply:
Can you also please remove inline Eslint override.